ci

```diff
@@ -66,4 +77,22 @@ case "${1:?}"-"${2:?}" in
         for f in `ls ${BuildDir}/_build/dev*/rel/epoch/log/epoch.log`; do echo \"\"; echo \"$f:\"; echo \"\"; cat $f; done
         for f in `ls ${BuildDir}/_build/dev*/rel/epoch/log/crash.log`; do echo \"\"; echo \"$f:\"; echo \"\"; cat $f; done
         ;;
+    deploy-package)
+        # Free Travis version does not support SSH key pairs, thus using encrypted files for SSH deploy key
+        # https://docs.travis-ci.com/user/encrypting-files/
+        openssl aes-256-cbc -K $encrypted_a2e547bcd39e_key -iv $encrypted_a2e547bcd39e_iv -in deployment/ansible/files/travis/deploy_key.enc -out /tmp/deploy_rsa -d
+        # ssh agent does not accept world readable keys
+        chmod 600 /tmp/deploy_rsa
+        eval "$(ssh-agent -s)"
+        ssh-add /tmp/deploy_rsa
+
+        BuildDir="${3:?}"
+        DeployEnv="${4:?}"
+        OsLabel=$(package_os_label "${TRAVIS_OS_NAME:?}")
+        TARBALL="${BuildDir:?}"/_build/prod/rel/epoch/epoch-"${Version:?}"-"${OsLabel:?}".tar.gz
```

Comment:
At line 62 `cp -p` -> `mv` lost in rebase. No change needed in this PR though.

Reply:
Ah, right. It will be overkill to wait travis again